src/impl.cpp

```diff
@@ -223,14 +223,17 @@ void Manifold::Impl::CreateFaces() {
   stable_sort(triPriority.begin(), triPriority.end(),
               [](auto a, auto b) { return a.area2 > b.area2; });
 
+  Vec<int> interiorHalfedges;
```

Comment:
It turns out the reallocation is quite significant. I wish the profiler can just give warning when there are many allocator calls within a certain period of time...